RTech.CoffeeMachine.Api/BrewCoffee/Util/IDateTimeProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RTech.CoffeeMachine.Api.BrewCoffee.Util;
+
+/// <summary>
+/// Represents an abstraction that can provide date+time.
+/// </summary>
+/// <remarks>
+/// Enhances code testability (interface can be mocked).
```

Comment:
I normally wouldnt put this comment in code (since it can be assumed), but I put it in anyways for clarity.